src/main/java/com/jvm_bloggers/ApplicationProfiles.java

```diff
@@ -2,6 +2,10 @@
 
 public class ApplicationProfiles {
 
+    private ApplicationProfiles() {
+        throw new IllegalStateException("Utility class");
+    }
+
```

Comment:
We have a Lombok dependency in the project already. We can use [`@UtilityClass`](https://projectlombok.org/features/experimental/UtilityClass) annotation instead.
```suggestion
@UtilityClass
public class ApplicationProfiles {
```

Reply:
Thank you. Done :)